parser/src/aspects/call.rs

```diff
@@ -75,16 +78,27 @@ impl<'a> CallAspect<'a> for Parser<'a> {
         let type_parameters = self.parse_type_parameter_list()?.0;
         if let Some(open_parenthesis) = self.cursor.advance(of_type(TokenType::RoundedLeftBracket))
         {
+            if identifier.token_type != Identifier {
+                return self.expected(
+                    "Expected function name.",
+                    ParseErrorKind::Expected("valid function identifier".to_string()),
+                );
```

Comment:
```suggestion
                return self.expected_with(
                    "Expected function name.",
                    identifier.clone(),
                    ParseErrorKind::UnexpectedInContext(format!(
                        "`{}` is a reserved keyword.",
                        identifier.value
                    )),
                );
```
This error is placed one token after the erroneous token, and doesn't state that the token is a keyword. I also have doubts about the placement of this code, as keywords are a more complex issue below the surface (see `echo while()`, `echo reef()` vs `reef()` behave quite differently).